### ISSUE TITLE
The menu only open from right

### ISSUE THREAD

**Issue by asifsaho** [steviebaa/mui-nested-menu#33]
the menu only opens from the right side. Sometimes the button is so close to the right that it is logical to see the submenu opening from the left side.
![Screenshot 2023-01-26 at 09 41 05](https://user-images.githubusercontent.com/2722232/214792232-78ecdc07-1e51-4280-8e37-32f0f23f8b61.png)

**Comment by mattspeller** [comment]
Yes a simple "left" prop would be great for those opening the nestedmenuitem from a navigation bar, from buttons docked to the right like in the screen shot above. In this case I'd always want the popups to appear to the left instead of right, as the popup itself is right: 0px

**Comment by sadikyalcin** [comment]
You can override the params. The components accept `MenuProps` which is the props of mui `Menu` compontent.

```
const menuProps = {
    anchorOrigin: {
      vertical: 'bottom',
      horizontal: 'left',
    },
    transformOrigin: {
      vertical: 'top',
      horizontal: 'right',
    },
  };

<NestedMenuItem
  ...
  MenuProps={menuProps}
>

```
